### ISSUE TITLE
Build failure with RN72 compileDebugAndroidTestKotlin (Android)

### ISSUE THREAD

**Issue by rjmatthews62** [Nozbe/WatermelonDB#1628]
Build is failing with `Could not create task ':nozbe_watermelondb:compileDebugAndroidTestKotlin`

Carefully following installation notes re: Kotlin version and @babel.

To recreate:
```
npx react-native init testprojectname
yarn add @nozbe/watermelondb
yarn add --dev @babel/plugin-proposal-decorators
(set .babelrc as per installation notes)
npx react-native run-android
```

build.gradle:
```
// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {
    ext {
        buildToolsVersion = "33.0.0"
        minSdkVersion = 21
        compileSdkVersion = 33
        targetSdkVersion = 33
        kotlinVersion = '1.3.50'
        // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
        ndkVersion = "23.1.7779620"
    }
    repositories {
        google()
        mavenCentral()
    }
    dependencies {
        classpath("com.android.tools.build:gradle")
        classpath("com.facebook.react:react-native-gradle-plugin")
    }
}
```

**Comment by mlecoq** [comment]
Did you try a more recent `kotlinVersion` ?

**Comment by rjmatthews62** [comment]
Yes, I tried 1.5.20 (IIRC) ... it would help if I had the slightest idea of what it was FOR and why the error was happening. I can find almost nothing about it online.

**Comment by sheggietyn** [comment]
> Build is failing with `Could not create task ':nozbe_watermelondb:compileDebugAndroidTestKotlin`
> 
> Edit: WatermelonDB 0.26.0 Edit: Windows 11
> 
> Carefully following installation notes re: Kotlin version and @babel.
> 
> To recreate:
> 
> ```
> npx react-native init testprojectname
> cd testprojectname
> yarn add @nozbe/watermelondb
> yarn add --dev @babel/plugin-proposal-decorators
> (set .babelrc as per installation notes)
> npx react-native run-android
> ```
> 
> build.gradle:
> 
> ```
> // Top-level build file where you can add configuration options common to all sub-projects/modules.
> 
> buildscript {
>     ext {
>         buildToolsVersion = "33.0.0"
>         minSdkVersion = 21
>         compileSdkVersion = 33
>         targetSdkVersion = 33
>         kotlinVersion = '1.3.50'
>         // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
>         ndkVersion = "23.1.7779620"
>     }
>     repositories {
>         google()
>         mavenCentral()
>     }
>     dependencies {
>         classpath("com.android.tools.build:gradle")
>         classpath("com.facebook.react:react-native-gradle-plugin")
>     }
> }
> ```

just add this to the top of the code in BundleHermesCTask.kt


 @OptIn(kotlin.ExperimentalStdlibApi::class)

**Comment by rjmatthews62** [comment]
> just add this to the top of the code in BundleHermesCTask.kt
> 
> @OptIn(kotlin.ExperimentalStdlibApi::class)

I don't suppose you have any detail on what this does or why it is needed?

**Comment by e1b2i3** [comment]
> Build is failing with `Could not create task ':nozbe_watermelondb:compileDebugAndroidTestKotlin`
> 
> Edit: WatermelonDB 0.26.0
> Edit: Windows 11
> 
> Carefully following installation notes re: Kotlin version and @babel.
> 
> To recreate:
> ```
> npx react-native init testprojectname
> cd testprojectname
> yarn add @nozbe/watermelondb
> yarn add --dev @babel/plugin-proposal-decorators
> (set .babelrc as per installation notes)
> npx react-native run-android
> ```
> 
> build.gradle:
> ```
> // Top-level build file where you can add configuration options common to all sub-projects/modules.
> 
> buildscript {
>     ext {
>         buildToolsVersion = "33.0.0"
>         minSdkVersion = 21
>         compileSdkVersion = 33
>         targetSdkVersion = 33
>         kotlinVersion = '1.3.50'
>         // We use NDK 23 which has both M1 support and is the side-by-side NDK version from AGP.
>         ndkVersion = "23.1.7779620"
>     }
>     repositories {
>         google()
>         mavenCentral()
>     }
>     dependencies {
>         classpath("com.android.tools.build:gradle")
>         classpath("com.facebook.react:react-native-gradle-plugin")
>     }
> }
> ```
